### PR TITLE
Fix Python 3.10

### PR DESCRIPTION
**Pull request by severinsimmler**
For some reason, the wheel for Python 3.10 is broken.

- Fix wheel for Python 3.10
- Remove some unnecessary files
- Remove compatibility stuff for Python 2
- Run tests against the wheels
- Formatting (Black, isort, pycln)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # jieba_pyfast
 
-A Chinese text segmentation module, from `jieba_fast`, with wheels for python3.9 & python3.10
+A Chinese text segmentation module, from `jieba_pyfast`, with wheels for python3.9 & python3.10
```
Comment (adam444555) [inline README.md]:
```suggestion
A Chinese text segmentation module, from `jieba_fast`, with wheels for python3.9 & python3.10
```

Comment (adam444555) [reply README.md]: This is a reference to the original library

**Review by adam444555 — COMMENTED** [review]
LGTM 👍  Just one small correct in README

File: README.md (hunk 1 above)
Comment (severinsimmler) [reply README.md]: :+1: :+1:

**Pull request merged**